docs/sources/administration/grafana-advisor/_index.md

```diff
@@ -96,6 +96,112 @@ If the [Grafana LLM app](https://grafana.com/grafana/plugins/grafana-llm-app/) i
 
 ![<Grafana Advisor - LLM suggestions>](/media/docs/grafana-advisor/llm-suggestions.png)
 
+## How to Create an Alert Based on Advisor Results
+
+This guide will walk you through creating a Grafana alert that monitors Advisor check results and triggers when failures are detected.
+
+## Prerequisites
+
+- Admin access to your Grafana instance
+
+## Step 1: Create a Service Account and Token
+
+1. Navigate to **Administration → Users and access → Service accounts** in your Grafana instance
+2. Click **Add service account**
+3. Provide a name (e.g., "advisor-alert-service-account")
+4. Set the role to **Admin** to ensure proper permissions
+5. Click **Create**
+6. In the service account details, click **Add service account token**
+7. Provide a token name and set an appropriate expiration
+8. Click **Generate token**
+
+> **Important**: Copy the token value immediately and store it securely - you won't be able to see it again
+
+## Step 2: Install Grafana Infinity plugin (if not already installed)
+
+1. Go to **Administration → Plugins**
```

Comment:
I think we can combine step 2 and 3 to make this simpler and do all through connections, i.e. Navigate to connections, search infinity, If not installed - click install, then click add new data source (from the plugin detail page)